xml/System.Reflection/Assembly.xml

```diff
@@ -5082,6 +5076,7 @@ The ability to execute code in remote assemblies is disabled. See <see href="htt
 
  ]]></format>
         </remarks>
+        <exception cref="T:System.NotSupportedException">.NET Core and .NET 5+ only: In all cases.</exception>
```

Comment:
Should we add this method to this page? https://learn.microsoft.com/en-us/dotnet/core/compatibility/unsupported-apis#systemreflection